pywinauto/timings.py

```diff
@@ -319,11 +319,45 @@ def WaitUntil(
             
     return func_val
 
+#=========================================================================
+def wait_until(

```

Comment:
I'm not sure that understood this comment right. How it should collaborate with first comment?

    def WaitUntill(timeout, 
               retry_interval, 
               func, 
               value = True, 
               op = operator.eq,
               *args):

    return wait_until(timeout, retry_interval, value, op)(func)(*args)